.github/actions/f3d-dependencies/action.yml

```diff
@@ -7,43 +7,46 @@ inputs:
     default: 'x86_64'
   alembic_version:
     description: 'Version of alembic to build'
-    required: true
+    required: false
   assimp_version:
     description: 'Version of assimp to build'
-    required: true
+    required: false
   draco_version:
     description: 'Version of draco to build'
-    required: true
+    required: false
   occt_version:
     description: 'Version of occt to build'
-    required: true
+    required: false
   openexr_version:
     description: 'Version of openexr to build'
-    required: true
+    required: false
   pybind11_version:
     description: 'Version of pybind11 to build'
-    required: true
+    required: false
   usd_version:
     description: 'Version of usd to build'
-    required: true
+    required: false
 
 runs:
   using: "composite"
   steps:
 
     - name: Install OCCT dependency
+      if: inputs.occt_version != ''
```

Comment:
it means that no version specified is equivalent to no dependency build?

Reply:
Yes, that's the point